bahmnicore-api/src/main/java/org/bahmni/module/bahmnicore/service/BahmniDrugOrderService.java

```diff
@@ -38,4 +39,13 @@ List<DrugOrder> getInactiveDrugOrders(String patientUuid, Set<Concept> concepts,
 
     List<BahmniDrugOrder> getDrugOrders(String patientUuid, Boolean isActive, Set<Concept> conceptsToFilter, Set<Concept> conceptsToExclude,
                                         String patientProgramUuid) throws ParseException;
+
+    List<BahmniDrugOrder> getBahmniDrugOrdersForVisit(String patientUuid, String visitUuid);
```

Comment:
this method too can be private. No need to expose as public or in the interface

Reply:
refactored as private method